src/python/system/shell.py

```diff
@@ -169,6 +169,11 @@ def _delete_object(path, delete_func):
     except:
       pass
 
+  if environment.get_value('SKIP_SYSTEM_TEMP_CLEANUP'):
```

Comment:
nit: might be moved to the very beginning of the function, but I guess it's just a style preference, feel free to ignore

Reply:
I think we followed this convention in other places, will keep as-is.